docs/app/components/grid/grid-demo.component.html

```diff
@@ -12,16 +12,17 @@ <h5>About</h5>
 
             <hc-tile>
                 <h5>Example </h5>
-                <table [hc-grid]="rows" [gridOptions]="gridOptions" #tableExample>
+                <table [hc-grid]="rows" [gridOptions]="gridOptions" class="hc-table-borders" #tableExample>
                     <thead>
                         <tr>
+                            <th hc-master-checkbox></th>
```

Comment:
Started out with wanting to include whether the rows should be selectable in the gridOptions, but wasn't sure of the best way to add the checkboxes in that instance or have access to which rows were selected.

Reply:
This is a tough problem as I don't think their is an easy answer. We could get together about some of the options...